File: csharp/gabaritos/gabarito15.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Digite o primeiro número: ");
        int a = int.Parse(Console.ReadLine());
        Console.Write("Digite o segundo número: ");
        int b = int.Parse(Console.ReadLine());
        while (b != 0)
        {
            int temp = b;
            b = a % b;
            a = temp;
        }
        Console.WriteLine($"MDC: {a}");
    }
}

File: cpp/gabaritos/gabarito15.cpp
#include <iostream>

int main() {
    int a, b;
    std::cout << "Digite o primeiro número: ";
    std::cin >> a;
    std::cout << "Digite o segundo número: ";
    std::cin >> b;
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    std::cout << "MDC: " << a << std::endl;
    return 0;
}